trigger-actions-framework/main/default/classes/TriggerActionFlowBypassProcessor.cls

```diff
@@ -0,0 +1,44 @@
+/*
+   Copyright 2022 Google LLC
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+public inherited sharing abstract class TriggerActionFlowBypassProcessor {
+	public static final Set<String> REQUEST_TYPES = new Set<String>{
```

Comment:
I think REQUEST_TYPES is already defined in TriggerActionConstants

Reply:
Good eye, cleaned this up